pkg/console/operator/sync_v400.go

```diff
@@ -53,49 +53,63 @@ func sync_v400(co *consoleOperator, originalOperatorConfig *operatorv1.Console,
 
 	rt, rtChanged, rtErr := SyncRoute(co, operatorConfig)
 	if rtErr != nil {
-		co.SyncStatus(co.ConditionResourceSyncFailure(operatorConfig, fmt.Sprintf("%v: %s\n", "route", rtErr)))
+		msg := fmt.Sprintf("%v: %s\n", "route", rtErr)
+		fmt.Printf("incomplete sync: %v \n", msg)
+		co.SyncStatus(co.ConditionResourceSyncIncomplete(operatorConfig, msg))
 		return operatorConfig, consoleConfig, toUpdate, rtErr
 	}
 	toUpdate = toUpdate || rtChanged
 
 	_, svcChanged, svcErr := SyncService(co, recorder, operatorConfig)
 	if svcErr != nil {
-		co.SyncStatus(co.ConditionResourceSyncFailure(operatorConfig, fmt.Sprintf("%q: %v\n", "service", svcErr)))
+		msg := fmt.Sprintf("%q: %v\n", "service", svcErr)
+		fmt.Printf("incomplete sync: %v \n", msg)
+		co.SyncStatus(co.ConditionResourceSyncIncomplete(operatorConfig, msg))
 		return operatorConfig, consoleConfig, toUpdate, svcErr
 	}
 	toUpdate = toUpdate || svcChanged
 
 	cm, cmChanged, cmErr := SyncConfigMap(co, recorder, operatorConfig, consoleConfig, infrastructureConfig, rt)
 	if cmErr != nil {
-		co.SyncStatus(co.ConditionResourceSyncFailure(operatorConfig, fmt.Sprintf("%q: %v\n", "configmap", cmErr)))
+		msg := fmt.Sprintf("%q: %v\n", "configmap", cmErr)
```

Comment:
These are a bit cumbersome, but `logging` + `status` seems the best way to ensure state is clearly known to someone attempting to debug.